Utilities/ITKv5Preparation/Move_DISALLOW_COPY_to_public_section.cpp

```diff
@@ -167,7 +167,7 @@ namespace
           }
           else
           {
-            if (StringStartsWithPrefix(c_str, "ITK_DISALLOW_COPY_AND_ASSIGN("))
+            if (StringStartsWithPrefix(c_str, "ITK_DISALLOW_COPY_AND_MOVE("))
```

Comment:
This should probably check both `ITK_DISALLOW_COPY_AND_ASSIGN` and `ITK_DISALLOW_COPY_AND_MOVE`.